question-service/migrations/20220910135944-create-category.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Categories', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      difficulty: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      types: {
+        type: Sequelize.ARRAY(Sequelize.STRING)
+      },
+      QuestionId: {
```

Comment:
I feel that question Id should still follow the same naming convention as the rest of the variables.